skfp/descriptors/constitutional.py

```diff
@@ -0,0 +1,278 @@
+from rdkit.Chem import AddHs, Mol
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumRings, CalcNumRotatableBonds
+
+
+def atom_count(mol: Mol, atom_symbol: str) -> int:
```

Comment:
1. Maybe `element_atom_count`?
2. I think it will be quite a bit more useful and elastic if you use `atom_id: Union[int, str]` instead, which could be either symbol or atomic number, and then select based on that.